config/initializers/backtrace_silencers.rb

```diff
@@ -0,0 +1,8 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Whats this file about?

Reply:
This is where we can add backtrace silencers for libraries that we're using but don't wish to see in our backtraces. `Rails.backtrace_cleaner.add_silencer { |line| /my_library/.match?(line) }` will skip any lines from my_library.